katas/square-digits/alejandra/squareDigits.js

```diff
@@ -1,3 +1,6 @@
 export function squareDigits(num) {
-    return 0;
```

Comment:
It's weird, since we already merge the PR where these changes were merged, I wouldn't expect these to appear in this PR. Although, I reviewed the branches that you created and those looks good, so there should not be a problem merge as is.

Reply:
That's odd. I don't know what happened but I'll try to find out. 
Thank you, Jose :)!